cachito/workers/pkg_managers/npm.py

```diff
@@ -68,6 +71,10 @@ def _get_deps(package_lock_deps, file_deps_allowlist, _name_to_deps=None):
         # output of this function to download the dependencies will ignore this dependency.
         if info["version"].startswith("file:") and name in file_deps_allowlist:
             log.info("The dependency %r is an allowed exception", info)
+        # If there is npmv7 workspace entry in dependencies of package-lock.json, skip it.
```

Comment:
Just a minor nitpick, but I'd leave the "v7" part out of this comment and the info log below. This is because workspaces are a permanent feature of NPM, even when v8, v9, etc. come around.

Reply:
right, good catch